app/build.gradle

```diff
@@ -20,7 +20,7 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
 
         Properties properties = new Properties()
-            properties.load(project.rootProject.file("local.properties").newDataInputStream())
+        properties.load(project.rootProject.file("local.properties").newDataInputStream())
```

Comment:
Deze tab had ik erin gezet omdat deze onder de properties valt en het met tab duidelijker was dat deze niet "losstaand" is.

Reply:
Reverted